src/alexaSkillDemo.py

```diff
@@ -0,0 +1,161 @@
+        shouldEndSession = True
+    else:
+        speechOutput = 'I am not sure what your building is. Please try again.' \
+                       'You can tell me your building by saying, my building is.'
+        shouldEndSession = False
+     
+    """Setting repromptText to None signifies that we do not want to reprompt
+       the user. If the user does not respond or says something that is not
+       understood, the session will end.
+    """   
+    return buildResponse(sessionAttributes, buildSpeechletResponse(
+        intent.get('name'), speechOutput, repromptText, shouldEndSession))
+
+""" --------------- Events ------------------ """
+def onSessionStarted(sessionStartedRequest, session):
+    """ Called when the session starts """
+    print('onSessionStarted requestId=' + sessionStartedRequest.get('requestId')
+          + ', sessionId=' + session.get('sessionId'))
+
+def onLaunch(launchRequest, session):
+    """ Called when the user launches the skill without specifying what they
+        want
+    """
+    print('onLaunch requestId=' + launchRequest.get('requestId') +
+          ', sessionId=' + session.get('sessionId'))
+    return getWelcomeResponse()
+
+def onIntent(intentRequest, session):
+    """ Called when the user specifies an intent for this skill """
+    
+    print('onIntent requestId=' + intentRequest.get('requestId') +
+          ', sessionId=' + session.get('sessionId'))
+
+    intent = intentRequest.get('intent')
+    intentName = intentRequest.get('intent').get('name')
+
+    """ Dispatch to skill's intent handlers """
+    if intentName == 'MyBuildingIsIntent':
+        return setBuildingInSession(intent, session)
+    elif intentName == 'WhatsMyBuildingIntent':
+        return getBuildingFromSession(intent, session)
+    elif intentName == 'AMAZON.HelpIntent':
+        return getWelcomeResponse()
+    elif intentName == 'AMAZON.CancelIntent' or intent_name == 'AMAZON.StopIntent':
+        return handleSessionEndRequest()
+    else:
+        raise ValueError('Invalid intent')
+
+def onSessionEnded(sessionEndedRequest, session):
+    """ Called when the user ends the session.
+        Is not called when the skill returns shouldEndSession=True
+    """
+    print('onSessionEnded requestId=' + sessionEndedRequest.get('requestId') +
+          ', sessionId=' + session.get('sessionId'))
+
+""" --------------- Main handler ------------------ """
+
+def lambdaHandler(event, context):
+    """ Route the incoming request based on type (LaunchRequest, IntentRequest,
+        etc.) The JSON body of the request is provided in the event parameter.
+    """
+    print('event.session.application.applicationId=' +
+          event.get('session').get('application').get('applicationId'))
+
+    """
+    if (event.get('session').get('application').get('applicationId') !=
+         'amzn1.echo-sdk-ams.app..get(unique-value-here)'):
+         raise ValueError('Invalid Application ID')
```

Comment:
I understand you probably commented this out for testing, but it's going to need to work for when you submit for certification later on. Check out https://developer.amazon.com/docs/custom-skills/certification-requirements-for-custom-skills.html if you haven't already.